mule-apikit-metadata/src/test/resources/org/mule/module/apikit/metadata/outputheaders-in-raml10/get-headers-router-config.out

```diff
@@ -0,0 +1,35 @@
+%type _:Java = (inputMetadata:{
+  "message" : @typeId("value" : "org.mule.runtime.api.message.Message") {
+    "payload" : Any, 
+    "attributes" : {
+      "queryParameters"? : {
+
+      }, 
+      "headers"? : {
+
+      }, 
+      "uriParameters"? : {
+
+      }
+    }
+  }, 
+  "variables" : {
+
+  }
+}) -> {
+  "message" : @typeId("value" : "org.mule.runtime.api.message.Message") {
+    "payload" : Any
+  }, 
+  "variables" : {
+    "outboundHeaders" : {
+      "stringHeader"? : @enum("values" : ["enum1","enum2"]) @length("min" : 2, "max" : 10) @default("value" : "enum1") String, 
+      "numberHeader"? : @range("from" : 10.1, "to" : 20.2) Number, 
+      "integerHeader"? : @range("from" : 10.0, "to" : 20.0) @int Number, 
+      "dateHeader"? : DateTime, 
+      "booleanHeader"? : Boolean, 
+      "fileHeader"? : Binary, 
+      "withoutType"? : String
+    }, 
```

Comment:
Like inboundheaders, default values are missing. Is this right?

Reply:
This is the behavior defined in metadata-model-raml metadata converter.